packages/react-environment/src/types/currency-config.interface.ts

```diff
@@ -28,6 +28,7 @@ export interface CurrencyView {
 }
 
 export interface CurrencyConfig extends CurrencyView {
```

Comment:
What are the advantages to having the chainType on both the CurrencyConfig and the ChainConfig? I think the ChainConfig should be sufficient

Reply:
removed